lib/stimulus_reflex/reflex.rb

```diff
@@ -111,6 +111,8 @@ def controller?
   end
 
   def render(*args)
+    options = args.extract_options!
+    args << options.reverse_merge!(layout: false)
```

Comment:
Isn’t the bang unnecessary here?

Reply:
It's possible that something was weird in my dev env, but my memory (it's been a few weeks) is that there was no "non-bang" version of this method available.

I'd be happy to be wrong...

Reply:
Haha, entirely possible 🤦‍♂️

Reply:
As a smart betting man would expect, you were right!

Great catch. 👍